### PR TITLE
Switch to Weblate for translations

### PR DESCRIPTION
**Pull request by yvanzo**
# Problem

Transifex is being replaced with Weblate for translations.

# Solution

The current setup is to cautiously make Weblate read `*.pot` files from the branch `beta` and push changes to `*.po` files to the branch `translations`. So the only manual step is to merge those to `beta` when appropriate.

These patches updates the docs for releasing, drops obsolete script, and changes links to point to our Weblate.

# Checklist for author

* [x] Adjusted private docs for releasing to follow these changes

### CODE REVIEW

**Review by reosarevok — CHANGES_REQUESTED** [review]
Seems good - one typo seems important to fix, but seems fine to merge otherwise.

File: INSTALL.md [hunk 1]
```diff
@@ -410,47 +410,28 @@ If you intend to run a server with translations, there are a few steps to follow
...
-    code, optionally followed by an underscore and an ISO country code (e.g. `fr`
-    for French, `fr_CA` for Canadian French).
+3.  Get the latest translations
 
-    Or, if you want to get _all_ translations instead:
+    Weblate is configured to follow the `*.pot` files from the branch `beta`
+    and to push its updated `*.po` files to to the branch `translations`.
 
-        tx pull -a
+    Either switch to the branch `translation` or merge it to your own branch.
```
[the remark this answers is not in the document]
Comment (yvanzo) [reply INSTALL.md:429]: Good catch
```suggestion
    Either switch to the branch `translations` or merge it to your own branch.
```

**Review by reosarevok — APPROVED** [review]

File: INSTALL.md [hunk 2]
```diff
@@ -410,47 +410,28 @@ If you intend to run a server with translations, there are a few steps to follow
...
-
-    This will download the .po files for your language(s) of choice to the po/
-    folder with the correct filenames. Languages are written as an ISO language
-    code, optionally followed by an underscore and an ISO country code (e.g. `fr`
-    for French, `fr_CA` for Canadian French).
+3.  Get the latest translations
 
-    Or, if you want to get _all_ translations instead:
+    Weblate is configured to follow the `*.pot` files from the branch `beta`
+    and to push its updated `*.po` files to to the branch `translations`.
```
Comment (mwiencek) [inline INSTALL.md:427]: Should we mention that we periodically merge the translations branch into beta/master?  (I'm not sure it's 100% clear that this step is unnecessary unless you want "bleeding edge" translations.)

**Review by mwiencek — APPROVED** [review]

File: INSTALL.md (hunk 2 above)
Comment (yvanzo) [reply INSTALL.md:427]: Indeed I assumed that this section was about working with “bleeding edge” translations, that’s why I added “the latest” to the title. I just clarified this point in the commit https://github.com/metabrainz/musicbrainz-server/pull/3024/commits/d08c9ae5c150e5a9b151f28d986e83da98d25f1a.

File: INSTALL.md [hunk 3]
```diff
@@ -421,12 +421,15 @@ If you intend to run a server with translations, there are a few steps to follow
...
-3.  Get the latest translations
+3.  Optionally, get the latest “bleeding edge” translations
 
     Weblate is configured to follow the `*.pot` files from the branch `beta`
     and to push its updated `*.po` files to to the branch `translations`.
+    Branches are regularly merged into each other, so translations will
+    be from the latest production release at least if not more recent.
 
-    Either switch to the branch `translations` or merge it to your own branch.
+    If you absolutely want the latest “bleeding edges” translations, then
```
Comment (reosarevok) [inline INSTALL.md:431]:
```suggestion
    If you absolutely want the latest “bleeding edge” translations, then
```

**Pull request merged**